Zeeguu Reader/Zeeguu Reader/AddFeedTableViewController.swift

```diff
@@ -102,6 +113,7 @@ class AddFeedTableViewController: UITableViewController {
 	}
 	
 	func cancel(sender: UIBarButtonItem) {
+		urlField.resignFirstResponder()
```

Comment:
do you really have to do this? 
if i implemented the framework, i would have done this for you :)